cloudbuild.yaml

```diff
@@ -17,11 +17,14 @@ steps:
     volumes: *vol
     args: ['go', 'build', './...']
     waitFor: ['untar-mod-cache']
+    env:
+      - 'GO111MODULE=on'
     id: 'build'
 
   - name: 'gcr.io/$PROJECT_ID/go-mod-ginkgo:0.1.5'
     volumes: *vol
     env:
+      - 'GO111MODULE=on'
```

Comment:
Formatting is off here, also you can update the version of the container to `0.2.1` for latest go version